### ISSUE TITLE
Downloads Page Cluttered, Install Instructions Not Prominent

### ISSUE THREAD

**Issue by jessebett** [JuliaLang/www.julialang.org#496]
The top of the Downloads page is cluttered, and makes it hard to find what is wanted: 
1. Downloads
2. Install Instructions

First, I disagree with @ViralBShah's suggestion on #364. I don't think the download button and blurb is in a good place on the download page. I understand that this might be a neccessity, but, @ViralBShah is there any way to see how many people are donating to Julia from this link? Seems like the cart before the horse putting the donate link before the downloads, I don't know much about this though.

https://github.com/JuliaLang/www.julialang.org/blob/e668266a8f7a9534ccb665b9a85889eeed928c1c/downloads/index.html#L14-L23

The next paragraph is (I expect) unlikely to be useful in most cases.

For one thing
>terminal using the built-in Julia command line using the binaries provided below

is a bit confusing. I imagine especially for beginners who aren't familiar with what "binaries" even means.

The next point, that you can use a Docker image.
To be honest, I'm not experienced with Docker. Would this make more sense in the platform instructions page? 
I feel like the top of this page should really be minded towards beginners who don't really know what they're looking for. 
People familiar with Docker images are more likely to search specifically for that right?
Again, not sure.

I like JuliaPro here, again, because I am in favour of this page being made easier for beginners.

https://github.com/JuliaLang/www.julialang.org/blob/e668266a8f7a9534ccb665b9a85889eeed928c1c/downloads/index.html#L27-L36

**Most importantly** the platform specific install instructions link is not great.
For one thing, these install instructions are on a completely different page, and I have some other thoughts on these, I may open an issue later.
But relevant to this page, it is not possible to search, e.g. for my platform `macOS` and get to instructions or even a link to those instructions from this page.

Possibly better, here, would be a list to the various platform instructions itemized by platform.

Further, it doesn't even finish the sentence before it suggests there will be problems installing Julia:
> Please see platform specific instructions for further installation instructions and if you have trouble installing Julia. If the provided download files do not work for you...

Let's be more optimistic, installing Julia is great! Not sure where these should be, but it clutters the most important information on this page for first time users, getting to install instructions.

https://github.com/JuliaLang/www.julialang.org/blob/e668266a8f7a9534ccb665b9a85889eeed928c1c/downloads/index.html#L39-L41

TL;DR:
Top of Downloads should be re-optimized with beginners in mind, and clear path to platform specific instructions.

**Comment by ViralBShah** [comment]
The install instructions are intentionally not put on the downloads page. This is because in the large majority of the cases, no instructions are needed. We provide standard installers for all the major OSes. Also the instructions are for non-standard installations, and often in some cases, unsupported binaries (provided by other distributions). I think we can move that entire paragraph about platform instructions and filing issues to the bottom. It actually is better placed with the description of support tiers. 

I have no strong feelings about the donate link. We obviously don't get huge donations, and even in what we get it is unclear how many are from there. I do feel strongly that asking for a github star has worked well, and I wouldn't want to change that.

**Comment by jessebett** [comment]
See the other issue for general comments on the role of the Downloads page and its priority relative to an Install Instructions page. I agree install instructions should not be put on the same page as downloads.

Strong disagree that the instructions should be below the downloads, again, see other thread about this.

Not against github star either.

**Comment by ViralBShah** [comment]
@jessebett Based on our follow-up chat, we have got the platform specific instructions in, and they are linked prominently in the downloads table. Setting up the PATH is also discussed. There's more work needed to make it even more accessible, but perhaps filing specific issues is what we need going forward.
